tox.ini

```diff
@@ -14,7 +14,14 @@ commands =
 deps =
     flake8==3.7.7
 
+[testenv:black]
+commands =
+    black . --check
+deps =
+    black>=19.3b0
+
 [tox]
 envlist =
     flake8
+    black
```

Comment:
Nit. Alphabetical order for consistency.

Reply:
It was changed too.